docs/src/solvers.md

```diff
@@ -95,3 +95,28 @@ JuMP.direct_model
 ```@docs
 JuMP.backend
 ```
+
+## Solver attributes
+
+Some solvers attributes can be queried through JuMP Models. Some solver attributes can setted through JuMP Models.
+
+```@docs
+solver_name
+```
+
+```@docs
+bridge_constraints
+```
+
+```@docs
+num_variables
+```
+
+```@docs
+set_silent
+```
+
+
+```@docs
```

Comment:
Why separate `@docs` blocks for `set_silent` and `unset_silent`?

Reply:
No special reason, I didn't realize that this
```julia
 @docs
set_silent
unset_silent
```
and this
```julia
@docs
set_silent
@docs
unset_silent
```
produce the same output